pkg/minikube/cluster/cluster.go

```diff
@@ -371,16 +373,32 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 }
 
 // GetHostLogs gets the openshift logs of the host VM.
-func GetHostLogs(api libmachine.API) (string, error) {
+// If follow is specified, it will tail the logs
+func GetHostLogs(api libmachine.API, follow bool) (string, error) {
 	host, err := CheckIfApiExistsAndLoad(api)
 	if err != nil {
 		return "", err
 	}
+
+	if follow {
```

Comment:
This really should be a if/else or two methods. It's a bit tricky to see the two returns and effective two different code paths